src/foam/nanos/auth/email/ServerEmailVerificationService.js

```diff
@@ -28,7 +28,8 @@
   ],
 
   constants: [
-    { name: 'TIMEOUT', type: 'Integer', value: 30 }
+    { name: 'TIMEOUT', type: 'Integer', value: 30 },
+    { name: 'VERIFY_EMAIL_TEMPLATE', type: 'String', value: 'verifyEmailByCode' }
```

Comment:
Yes, better as constants then magic values, but really this should be 'properties' to be configured.  Ok, for now, but thoughts on how each vendor/spid can be configured differently? 
Templates can be spid specific - so perhaps this is sufficient.